core/client/src/main/java/alluxio/client/file/options/CompleteFileOptions.java

```diff
@@ -53,12 +53,28 @@ public CompleteFileOptions setUfsLength(long ufsLength) {
     return this;
   }
 
-  /**
-   * @return the name : value pairs for all the fields
-   */
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof CompleteFileOptions)) {
+      return false;
+    }
+    CompleteFileOptions that = (CompleteFileOptions) o;
+    return Objects.equal(mUfsLength, that.mUfsLength);
```

Comment:
This case works because (null instanceof X) is always false. It will take some time to cover all of these methods, do you mind if I do so in a followup PR?